comeet/Data/Extensions/Date+API.swift

```diff
@@ -12,24 +12,28 @@ extension Date {
     
     static func fromAPI(stringDate: String) -> Date? {
         let formatter = DateFormatter()
+        formatter.timeZone = TimeZone(abbreviation: "UTC")
         formatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ss-SSSS"
```

Comment:
Is that still true?  All input date parameters use the same parser as of 
https://github.com/bookaroom/RESTApi/commit/21b9e4676c2d3cd9e75266d2d13e4a5dae1fb54b
(mostly implemented in https://github.com/bookaroom/RESTApi/commit/73db575100d68ced6166d16f6f1df805e91b5600)